pkg/runtime/service_controller.go

```diff
@@ -258,18 +258,22 @@ func (c *serviceController) BindControllerManager(mgr ctrlrt.Manager, cfg ackcfg
 		}
 	}
 
-	exporterInstalled, err := c.GetFieldExportInstalled(mgr)
+	exporterInstalled := false
```

Comment:
nit:
```suggestion
	exporterEnabled := false
```
you can then set it to True inside the if/else